plugins/content/caption/search/CaptionSearchPlugin.php

```diff
@@ -252,7 +254,8 @@ protected static function getElasticLines($items, $language)
 
 			$content = kString::stripUtf8InvalidChars($content);
 			$content = kXml::stripXMLInvalidChars($content);
-			$content= substr($content, 0 , kElasticSearchManager::MAX_LENGTH);
+			if(strlen($content))
```

Comment:
If already called strlen , check size not bigger than  kElasticSearchManager::MAX_LENGTH